src/java.desktop/macosx/classes/sun/lwawt/macosx/LWCToolkit.java

```diff
@@ -204,14 +204,16 @@ public LWCToolkit() {
     /*
      * System colors with default initial values, overwritten by toolkit if system values differ and are available.
      */
-    private static final int NUM_APPLE_COLORS = 3;
+    private static final int NUM_APPLE_COLORS = 4;
     public static final int KEYBOARD_FOCUS_COLOR = 0;
     public static final int INACTIVE_SELECTION_BACKGROUND_COLOR = 1;
     public static final int INACTIVE_SELECTION_FOREGROUND_COLOR = 2;
+    public static final int CONTROL_TEXT_COLOR = 3;
     private static int[] appleColors = {
         0xFF808080, // keyboardFocusColor = Color.gray;
         0xFFC0C0C0, // secondarySelectedControlColor
         0xFF303030, // controlDarkShadowColor
+        0xFFFFFFFF, // controlTextColor
```

Comment:
As far as I understand the fix, It is a "alternateSelectedControlTextColor" not an "controlTextColor"

Reply:
BTW isn't it is the same color as java_awt_SystemColor_CONTROL_LT_HIGHLIGHT?